source/Makefile.Objects

```diff
@@ -727,3 +729,8 @@ OBJS_TENSOR=tensor.o\
     lr_spectrum.o\
     hamilt_casida.o\
     esolver_lrtd_lcao.o\
+
+OBJS_RDMFT=rdmft.o\
+        rdmft_tools.o\
+        cal_V_rdmft.o\
```

Comment:
The comment here was in response to the flowchart, but apparently I got it in the wrong place again...
sorry, sorry!

Reply:
no, I mean, if you can only name the contribution term from dmft as "potential", then you can use term "pot_rdmft" as the file name. However, if the potential can be named more exactly, then please choose the best one.

Reply:
Thank you, your suggestion and motivation are very clear. I have changed the file name to rdmft_cal_hamilt.cpp, but I suddenly realized that this may not be so appropriate.

Usually in KS-DFT, we add up the potential matrices to get the effective Hamiltonian H^KS of the KS system (that is, Hamilt_mat = V_T + V_local + V_hartree + ... as I wrote above), but in rdmft, there is no such universal and concise effective Hamiltonian H^RDMFT. For example, when using the CG method of Riemann space to optimize {φi}, we do not need to construct an effective Hamiltonian; when using an iterative diagonalization method, we need to artificially construct a Fock-like matrix (that is, the F matrix in the flowchart I posted), which is not a simple addition of V_T and V_hartree, and its eigenvalues ​​do not have the meaning of energy levels. So, from these perspectives, it is not appropriate to change the file name to hamilt.

I think the names of V_hartree and V_local are clear enough and consistent with the common symbols of physical formulas for each contribution term, and I want to continue using them. So should I continue to use the name rdmft_cal_V.cpp and add more detailed comments to the function declaration, or use the name rdmft_cal_pot.cpp?

Or do you have any other suggestions?

Reply:
Thank you, your suggestion and motivation are very clear. I have changed the file name to rdmft_cal_hamilt.cpp, but I suddenly realized that this may not be so appropriate.

Usually in KS-DFT, we add up the potential matrices to get the effective Hamiltonian H^KS of the KS system (that is, Hamilt_mat = V_T + V_local + V_hartree + ... as I wrote above), but in rdmft, there is no such universal and concise effective Hamiltonian H^RDMFT. For example, when using the CG method of Riemann space to optimize {φi}, we do not need to construct an effective Hamiltonian; when using an iterative diagonalization method, we need to artificially construct a Fock-like matrix (that is, the F matrix in the flowchart I posted), which is not a simple addition of V_T and V_hartree, and its eigenvalues ​​do not have the meaning of energy levels. So, from these perspectives, it is not appropriate to change the file name to hamilt.

I think the names of V_hartree and V_local are clear enough and consistent with the common symbols of physical formulas for each contribution term, and I want to continue using them. So should I continue to use the name rdmft_cal_V.cpp and add more detailed comments at the beginning of the file, or use the name rdmft_cal_pot.cpp?

Or do you have any other suggestions?

Reply:
hmmm...I am not sure. 
For the file name, I think "cal_" always appears in the function name, which indicates "this is the function **calculates** something". This is the reason why I suggest the term "pot_rdmft", you can place several related implementation in this file, and there will be not any misleading words.

For the concept of "potential", there is a folder named source/module_elecstate/potentials, several potential implementations are placed there. Could you check if that fits your case? In that folder, you will find there are files named starting with "pot_".

But you are the one knows the most about your codes, so you can make your decision.

Reply:
What you said makes sense, I will delete "cal_".

The potential under source/module_elecstate/potentials seems to be the potential obtained by grid integration, which is consistent with what I have here in the physical sense, but I want to emphasize that under the NAOs representation, perhaps the name rdmft_pot_mat can be used

Reply:
wait, mat is the data structure, it would be better not include it in file name

Reply:
OK, I use rdmft_pot.cpp to instead